packages/framework/src/Resources/config/services.yaml

```diff
@@ -1039,3 +1039,5 @@ services:
     Shopsys\FrameworkBundle\Component\Messenger\MessageDispatcherDependency:
         arguments:
             $transportDsn: '%env(MESSENGER_TRANSPORT_DSN)%'
+
+    Shopsys\FrameworkBundle\Component\DateTimeHelper\DateTimeHelper: ~
```

Comment:
I would consider adding `Helper` suffix to the autoconfiguration part. Along with `Shopsys\FrameworkBundle\Model\Heureka\HeurekaShopCertificationLocaleHelper`, there are two helpers now in the `services.yaml` file :slightly_smiling_face:

Reply:
as we discussed, helper may not be a service